ops/lost-in-translation.py

```diff
@@ -0,0 +1,79 @@
+from subprocess import call, check_output, STDOUT
+from json import loads as decode_json, JSONDecodeError
+
+'''
+   This script finds all the english translation keys across
+   all commits (including branches and pull requests) for each
+   locale.json in app/locales. Afterwards it looks at each locale
+   separately to see if it's missing any one the english keys.
+
+   Usage: python3 ops/lost-in-translation.py
+   Important: You need to run it from the root directory.
+'''
+
+def git(*args):
+    '''Calls git cli with arguments.'''
+    stdout = check_output(['git', *args], stderr=STDOUT)
+    return stdout
+
+def find_commit_hashes_for_file(file_path):
+    '''Returns the commit hashes for a file path across all local branches.'''
+    lines = git('log', '--pretty=format:"%h"', '--all', '--', file_path)
+    hashes = [line.replace('"', '') for line in lines.decode('utf-8').split('\n')]
+    return hashes
+
+def retrieve_json_for_file_at_commit_hash(file_path, commit_hash):
+   '''Returns the JSON representation of the contents of a file at certain commit hash.'''
+   contents_at_commit_hash = git('show', '%s:%s' % (commit_hash, file_path))
+   try:
+       json = decode_json(contents_at_commit_hash.decode('utf-8'))
+       return json
+   except JSONDecodeError:
+       # Sometimes the file is not in a proper JSON format. We simply return nothing in that case.
+       return {}
+
+if __name__ == '__main__':
+    all_locales = ('no', 'se', 'en-IN')
```

Comment:
Is this correct?

Reply:
Just to test, but should include all locales.

Reply:
Would be nice to import that from all available locales in our repo.

Reply:
Very good point – I'll add that